src/services/TestDataService.ts

```diff
@@ -2157,4 +2158,44 @@ export class TestDataService {
 
         return Object.assign({}, basicCmsPage, overrides);
     }
+
+    getBasicCustomerGroup(overrides: Partial<CustomerGroup>): Partial<CustomerGroup> {
+        const customerGroupUuid = this.IdProvider.getIdPair().uuid;
+        const customerGroupName = `${this.namePrefix}CustomerGroup-${customerGroupUuid}${this.nameSuffix}`;
+
+        const basicCustomerGroup = {
+            id: customerGroupUuid,
+            name: customerGroupName,
+            displayGross: true,
+            registrationActive: true,
+            registrationTitle: customerGroupName,
+            registrationIntroduction: `${customerGroupName}-Introduction`,
+            registrationSeoMetaDescription: `${customerGroupName}-SEO-Description`,
+            registrationOnlyCompanyRegistration: false,
+            customFields: {},
+            registrationSalesChannels: [{
+                id: this.defaultSalesChannel.id,
+            }],
+
+        };
+
+        return Object.assign({}, basicCustomerGroup, overrides);
+    }
+
+    async createCustomerGroup(overrides: Partial<CustomerGroup>): Promise<CustomerGroup> {
```

Comment:
```suggestion
    async createCustomerGroup(overrides: Partial<CustomerGroup> = {}): Promise<CustomerGroup> {
```

Define here an empty override property by default so that it is not necessary to use the override

Reply:
Applied